assets/js/pages/orgs.tsx

```diff
@@ -22,9 +22,9 @@ const OrgList: React.FC = () => {
                     <span>{org.repo_count} Repos</span>
                 </Col>
                 <Col className="org-location">
-                {org.location &&
+                {org.yml_data.location &&
```

Comment:
Shouldn't be `{ org.yml_data.location || org.location }` here too?

Reply:
I let mandatory from the yml_data because gitlab does not retrieve location

Reply:
I would reverse the precedence anyways: `{ org.location || org.yml_data.location }`

Reply:
In that case for gitlab will be appearing the yml location and for github the org.location, since github retrieves that field, it's confusing...